tools/metricsdocs/metricsdocs_suite_test.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestNetwork(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "bumper Suite")
```

Comment:
copy paste, please change to metrics suit

Reply:
updated